libnmstate/netapplier.py

```diff
@@ -156,7 +157,15 @@ def _apply_ifaces_state(
                     con_profiles=ifaces_add_configs + ifaces_edit_configs,
                 )
             if verify_change:
-                _verify_change(desired_state)
+                for _ in range(5):
+                    try:
+                        _verify_change(desired_state)
+                        break
+                    except NmstateVerificationError:
+                        time.sleep(1)
```

Comment:
Use constant like `VERIFY_RETRY_INTERNAL` instead of magic number.

Reply:
Added. Thanks!

Reply:
`VERIFY_RETRY_INTERNAL` is 1 based on your code. and `VERIFY_RETRY_TIMEOUT` is 5.

Reply:
Ops. Done, thank you!